index.md

```diff
@@ -6,13 +6,17 @@ is_home: true
 
 <p class="lead">Web sites are made of lots of things — frameworks, libraries, assets, and utilities. Bower manages all these things for you.</p>
 
+Keeping track of all these packages and making sure they are up to date (or set to the specific versions you need) is tricky. Bower to the rescue!
+
+Bower can manage components that contain HTML, CSS, JavaScript, fonts or even image files. Bower doesn't concatenate or minify code or do anything else - it just installs the right versions of the packages you need and their dependencies.
+
 To [get started](#getting-started), Bower works by fetching and installing [packages](/search) from all over, taking care of hunting, finding, downloading, and saving the stuff you're looking for. Bower keeps track of these packages in a manifest file, [`bower.json`](/docs/creating-packages/#bowerjson). How you use [packages](/search) is up to you. Bower provides hooks to facilitate using packages in your [tools and workflows](/docs/tools).
 
-Bower is optimized for the front-end. Bower uses a flat dependency tree, requiring only one version for each package, reducing page load to a minimum.
+Bower is optimized for the front-end. If a package (let's use jQuery as an example) has multiple packages depending on it, Bower will download jQuery just once. This is known as a flat dependency graph and it helps reduce page load. (NPM, on the other hand, will download a version of jQuery for every package that depends on it).
 
 ## Install Bower
 
-Bower is a command line utility. Install it with npm.
+Bower is a command line utility. Install it globally with npm.
```

Comment:
Nit pick: "globally" doesn't improve this sentence IMO. Those who don't know the command line might be scared off by a "global" installation. Also Bower works just as well on a local config.